projects/storefrontlib/cms-components/content/video/video.component.html

```diff
@@ -0,0 +1,25 @@
+<div
+  *ngIf="data$ | async as data; else loading"
+  class="video-container"
+  tabindex="-1"
+>
+  <a *ngIf="data.overlayTitle" [routerLink]="routerLink">{{
+    data.overlayTitle
+  }}</a>
```

Comment:
It was added to the overlayTitle because on Safari, you need to begin the video by clicking on the center of the screen which triggered the routing.

We will modify the player in the next iteration when we add closed captions